### ISSUE TITLE
Convert Jupyter notebooks

### ISSUE THREAD

**Issue by esafak** [pystitch/stitch#24]
It would be nice if stitch could ingest notebooks so I can do my work in Jupyter then use stitch for archiving and converting.

**Comment by TomAugspurger** [comment]
Agreed. [This](http://tomaugspurger.github.io/pages/timeseries.html) example started as a notebook. To get to the HTML, I

- cleared all the cell output
- ran it through nbconvert to markdown
- replaced all the opening

    ```python
    ```

tags with

    ```{python}
    ```

Not ideal :(  I'm hoping that [podoc](https://github.com/podoc/podoc/) or a project like it will come along. That would convert from notebook to and from pandoc's JSON AST, which is where stitch does all its work.

**Comment by TomAugspurger** [comment]
Thinking about this today, we could provide an nbconver template that hides output and converts code-chunks from 

    ```
    code
    ```

to

    ```{kernel-name}
    code
    ```

That would help a bit.

**Comment by janschulz** [comment]
It would be great if you cout make the first part configgureable:

```
   "python": <kernel-name>
   "python3": <different-kernel-name>

...
` ` `{python}
<code>
` ` `
```

That way it can be reused in normal settings (and I really need that because e.g. I have no `r` kernel, only `ir-something`...

**Comment by grst** [comment]
I recently developed an Rmarkdown plugin for the [ipymd project](https://github.com/grst/ipymd), which adresses this problem. It allows to directly open and edit Rmarkdown (or pymarkdown...) documents from jupyter, entirely replacing the `ipynb` notebooks. 

I would still consider it in a "beta" state, but for now, it works quite well for me.
